doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -1181,3 +1182,33 @@
     };
 
     ```
+
+53. `iox::roudi::RouDiEnvironment` is moved to `iox::roudi_env::RouDiEnv` and in a separate library
+
+    There is still an alias on the old location with a deprecation warning. The API also changed a bit.
+    ```cpp
+    // before
+    #include "iceoryx_posh/testing/roudi_environment/roudi_environment.hpp"
+    iox::roudi::RouDiEnvironment roudiEnv{/*config*/};
+
+    // after
+    #include "iceoryx_posh/roudi_env/roudi_environment.hpp"
+    iox::roudi_enf::RouDiEnv roudiEnv{/*config*/};
+
+
+    // before
+    roudiEnv.InterOpWait();
+
+    // after
+    roudiEnv.triggerDiscoveryLoopAndWaitToFinish();
+
+
+    // before
+    SetInterOpWaitingTime(/*chrono*/);
```

Comment:
Why not `roudiEnv.SetInterOpWaitingTime(/*chrono*/)`? I haven't looked it up yet, but I'd expect this is also a method of the roudi environment.

Reply:
No, I have deprecated `InterOpWait`/`SetInterOpWaitingTime` and replaced them with `triggerDiscoveryLoopAndWaitToFinish`/`setDiscoveryLoopWaitToFinishTimeout`. The former did not fully state when it is needed and I found some cases where `InterOpWait` was added in tests without being required. I hope the more expressive names make it more clear when and especially when not to use

Reply:
Maybe a misunderstanding. I meant that I'd expected the (old and new) method to be called on the `roudiEnv` instance, like `roudiEnv.method()`.

Reply:
Oh, sure :sweat_smile: 

Depending on what one uses it is either `roudiEnv.method()` or  when `RouDi_GTest` is used just `method`.

Reply:
... and you are right that for consistency reasons it should mirror `triggerDiscoveryLoopAndWaitToFinish`